### PR TITLE
Fixes to superbuild on mac and linux after agents failed when running on multiple branches

### PR DESCRIPTION
**Pull request by PhucNgo1711**
Updates from root master

Fixes issue #(Enter issue number here)

## Proposed Changes
  -
  -
  -

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1120,7 +1124,156 @@ jobs:
...
+      inputs:
+        pathtoPublish: '$(Build.ArtifactStagingDirectory)'
+        artifactName: '$(Build.DefinitionName)_installer_Linux_ubuntu1604'
+        continueOnError: true
+
+    - task: DownloadBuildArtifacts@0
+      inputs:
+        buildType: 'current'
+        downloadType: 'single'
+        artifactName: '$(Build.DefinitionName)_installer_Linux_ubuntu1604'
```
Comment (AlexanderGetka-cbica) [inline azure-pipelines.yml:1260]: These two artifacts should be labelled as centos7 rather than ubuntu.

Comment (sarthakpati) [reply azure-pipelines.yml:1260]: Could the installers be named as ```$(Build.DefinitionName)_Installer_centos7```?

Comment (PhucNgo1711) [reply azure-pipelines.yml:1260]: My bad. Will add that in.

**Comment by AlexanderGetka-cbica** [comment]
What is the status on this?

**Comment by PhucNgo1711** [comment]
> What is the status on this?

I'm running into some issues with the superbuild on the cluster. I'm working with Mark on this.

**Pull request merged**